demo-app/package.json

```diff
@@ -39,7 +39,6 @@
     "@babel/polyfill": "^7.0.0",
     "@webcomponents/custom-elements": "^1.0.9",
     "copy-webpack-plugin": "^4.5.2",
-    "core-js": "^2.5.7",
```

Comment:
Trying to remember my rationale on why I directly added this.  We certainly get it transitively from @babel/polyfill.  Maybe I was thinking it would be obvious that we can use es6 features in this baseline?

Assuming it will come in fine transitively.  Did it run on ie11 ok?

Reply:
Yeap, everything works in IE still